From b05ea072859f9e64da5188726ad2d44db8706ace Mon Sep 17 00:00:00 2001
From: Sebastian Reichel <sebastian.reichel@collabora.com>
Date: Wed, 8 Oct 2025 18:15:24 +0200
Subject: [PATCH 084/177] arm64: dts: rockchip: add missing UFS regulators to
 ROCK 4D

Add missing regulator information for the ROCK 4D UFS interface.

Signed-off-by: Sebastian Reichel <sebastian.reichel@collabora.com>
---
 arch/arm64/boot/dts/rockchip/rk3576-rock-4d.dts | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3576-rock-4d.dts b/arch/arm64/boot/dts/rockchip/rk3576-rock-4d.dts
index b338c63db..2a69bd310 100644
--- a/arch/arm64/boot/dts/rockchip/rk3576-rock-4d.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3576-rock-4d.dts
@@ -930,6 +930,9 @@ &uart0 {
 };
 
 &ufshc {
+	vcc-supply = <&vcc_3v3_s0>;
+	vccq-supply = <&vcc_1v2_ufs_vccq_s0>;
+	vccq2-supply = <&vcc_1v8_ufs_vccq2_s0>;
 	status = "okay";
 };
 
-- 
2.47.3

